code/modules/religion/sparring/ceremonial_gear.dm

```diff
@@ -0,0 +1,42 @@
+///ritual weapons. they're really bad, but they become normal weapons when sparring.
+/obj/item/ceremonial_blade
+	name = "ceremonial blade"
+	desc = "A blade created to spar with. It seems weak, but if you spar with it...?"
+	icon_state = "default"
+	inhand_icon_state = "default"
+	icon = 'icons/obj/items/ritual_weapon.dmi'
+
+	//does the exact thing we want so heck why not
+	greyscale_config = /datum/greyscale_config/ceremonial_blade
+	greyscale_config_inhand_left = /datum/greyscale_config/ceremonial_blade_lefthand
+	greyscale_config_inhand_right = /datum/greyscale_config/ceremonial_blade_righthand
+	greyscale_colors = "#FFFFFF"
+
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	custom_materials = list(/datum/material/iron = 12000)  //Defaults to an Iron blade.
+	force = 2 //20
+	throwforce = 1 //10
+	w_class = WEIGHT_CLASS_NORMAL
+	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "cuts")
+	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "cut")
+	block_chance = 50
```

Comment:
free riot shield+, nice

Reply:
you're right, this should totally be a sparring stat